pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1600,3 +1600,29 @@ func TestMetricsReporter(t *testing.T) {
 	metricstest.CheckLastValueData(t, "pending_pods", wantTags, 1996)
 	metricstest.CheckLastValueData(t, "terminating_pods", wantTags, 1955)
 }
+
+func TestResolveScrapeTarget(t *testing.T) {
+	pa := kpa(testNamespace, testRevision)
+	tc := &testConfigStore{config: defaultConfig()}
+
+	want := "echo"
+	got := resolveScrapeTarget(tc.ToContext(context.Background()), pa, "echo")
+	if got != want {
+		t.Errorf("reconcileMetricSN()= %v, want %v", got, want)
+	}
```

Comment:
```suggestion
	if got, want := resolveScrapeTarget(tc.ToContext(context.Background()), pa, "echo"), "echo"; got != want {
		t.Errorf("reconcileMetricSN()= %s, want %s", got, want)
	}
```

Reply:
Same below.

Reply:
Fixed in 7b1feced1c20f12f662502628344d8087c3978f6.